xpk.py

```diff
@@ -149,6 +305,30 @@
     {machine_label}
 ---
 apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
```

Comment:
These resource flavors are PW specific. We shouldn't apply them when the cluster is not using pathways right?

Reply:
Agreed, I already moved to a helper in the new changes! Thank you